src/main/java/org/junit/internal/Checks.java

```diff
@@ -0,0 +1,37 @@
+package org.junit.internal;
+
+/** @since 4.13 */
+public final class Checks {
```

Comment:
As you can see in the commits, we have multiple classes that do `null` checks. Quite often when I write code in JUnit I find myself wanting a one-liner to check for null.

I doubt anyone would depend on this class, since 1) it's in an internal package, 2) it has an odd name, and 3) most everyone that is using JDK 6 or later would use `Objects.requireNotNull()` or Guava's `Preconditions.checkNotNull()`.